viadot/tasks/vid_club.py

```diff
@@ -115,5 +120,17 @@ def run(
             region=region,
             days_interval=days_interval,
         )
+        if cols_to_drop is not None:
+            if type(cols_to_drop) is list:
```

Comment:
You can use such a structure to write in a more pythonic way:
```suggestion
            if isinstance(cols_to_drop, list):
```

Reply:
ok